src/Aspire.Dashboard/Authentication/Connection/ListenOptionsConnectionTypeExtensions.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+
+namespace Aspire.Dashboard.Authentication.Connection;
+
+internal static class ListenOptionsConnectionTypeExtensions
+{
+    public static void UseConnectionType(this ListenOptions listenOptions, ConnectionType[] connectionTypes)
```

Comment:
`UseConnectionTypes` (plural) feels more natural to me at call sites, given it takes multiple types.

Reply:
```suggestion
    public static void UseConnectionTypes(this ListenOptions listenOptions, ConnectionType[] connectionTypes)
```